dm/tests/ha_cases/run.sh

```diff
@@ -138,7 +138,8 @@ function test_standalone_running() {
 	# test running, test2 fail
 	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
 		"query-status" \
-		"\"taskStatus\": \"Running\"" 1
+		"\"stage\": \"Running\"" 1 \
```

Comment:
Running task and offline worker looks strange.

Reply:
the `running : 1`  is subtask running on the online worker1

the query-status response looks like this
```
{
    "result": true,
    "msg": "",
    "sources": [
        {
            "result": true,
            "msg": "",
            "sourceStatus": {
                "source": "mysql-replica-01",
                "worker": "worker1",
                "result": null,
                "relayStatus": null
            },
            "subTaskStatus": [
                {
                    "name": "test",
                    "stage": "Running",
                    "unit": "Sync",
                    "result": null,
                    "unresolvedDDLLockID": "",
                    "sync": {
                        "totalEvents": "27",
                        "totalTps": "2",
                        "recentTps": "0",
                        "masterBinlog": "(mysql-bin.000002, 5693)",
                        "masterBinlogGtid": "cd3cf208-edc0-11eb-852c-d674c270ea47:1-26",
                        "syncerBinlog": "(mysql-bin.000002, 5387)",
                        "syncerBinlogGtid": "",
                        "blockingDDLs": [
                        ],
                        "unresolvedGroups": [
                        ],
                        "synced": false,
                        "binlogType": "remote",
                        "secondsBehindMaster": "0"
                    }
                }
            ]
        },
        {
            "result": false,
            "msg": "can't find task test2 from dm-worker, please use dmctl list-member to check if worker is offline.",
            "sourceStatus": {
                "source": "mysql-replica-02",
                "worker": "unknown-offline-worker",
                "result": null,
                "relayStatus": null
            },
            "subTaskStatus": [
                {
                    "name": "test2",
                    "stage": "InvalidStage",
                    "unit": "InvalidUnit",
                    "result": null,
                    "unresolvedDDLLockID": ""
                }
            ]
        }
    ]
}
```

Reply:
`query-status` should keep a simple output, above output should be `query-status --more`

Reply:
ok,  should i make the resp like this ?
```
{
    "result": true,
    "msg": "",
    "tasks": [
        {
            "taskName": "test-1",
            "taskStatus": "Running",
            "sources": [
                "mysql-01"
            ]
        },
        {
            "taskName": "test-2",
            "taskStatus": "InvalidStage",
            "sources": [
                "mysql-01"
            ]
            "msg": "can't find task test2 from dm-worker, please use dmctl list-member to check if worker is offline.",
        }
    ]
}
```

Reply:
sgtm

Reply:
> // if any result is false, we still print the full status.

i found this [commnet](https://github.com/pingcap/ticdc/blob/ca628c6f4d3831ad31c074c393253129b27174da/dm/dm/ctl/master/query_status.go#L94), so seems show full staus when some result=false is by design? 😅

Reply:
seems it's not a easy change for `query-status --more`, current output is OK to me

Reply:
> > // if any result is false, we still print the full status.
> 
> i found this [commnet](https://github.com/pingcap/ticdc/blob/ca628c6f4d3831ad31c074c393253129b27174da/dm/dm/ctl/master/query_status.go#L94), so seems show full staus when some result=false is by design? 😅

Yes, it's by design. I think we can do this in another PR after a further discussion if needed.